src/apps/user/templates/user/user_detail.html

```diff
@@ -1,11 +1,11 @@
 {% extends "base.html" %}
 
 {% block meta %}
```

Comment:
lets extend head meta blocks

look here we have to write <title> every time we create page

Can we think of some solution to avoid it?

Reply:
i have such proposition: pass context["title"] from each view.
it will be used for page title in body as well as for meta
what do you think?

Reply:
take a look at <head></head>
can we create title block?

<head>
  <title>{% block meta_title %}{% endblock %}</title>
</head>

in that case we can simply set any name we want

Reply:
so, the problem, that we write <title> tag each time?
we can definitely include it in base html and only write the name